### PR TITLE
🚀 Release v1.9.0

### PR DESCRIPTION
**Pull request by chandankumar4**
**What this PR does / why we need it**:

- Release notes for v1.9.0

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Part of: https://github.com/kubernetes-sigs/cluster-api/issues/11092

/area release

### CODE REVIEW

File: CHANGELOG/v1.9.0.md [hunk 1]
```diff
@@ -0,0 +1,448 @@
...
+- MachineHealthCheck: Increase timeout for 2 MHC unit tests (#11019)
+- MachinePool: Improve TestReconcileMachinePoolMachines unit test (#11124)
+- MachineSet: Add MD watch to the MS controller (#11448)
+- MachineSet: Extend MS ScalingUp and Remediationg conditions to include preflight check errors (#11390)
+- MachineSet: Implement MS remediating conditions (#11382)
+- Metrics: Add nilIsZero to all KSM metric configs where needed (#11101)
+- Metrics: Revert "Add nilIsZero to all KSM metric configs where needed" (#11107)
+- ClusterClass/MachinePool: Remove paused handling from reconcileExternal (#11400)
+- KCP/MachineSet: Implement KCP and MS Deleting conditions (#11381)
+- Machine/KCP]: Implement grace period for KCP remote conditions (#11339)
```
Comment (Sunnatillo) [inline CHANGELOG/v1.9.0.md:283]: Please remove "]" 
```suggestion
- Machine/KCP: Implement grace period for KCP remote conditions (#11339)
```

**Comment by chrischdi** [comment]
/approve

Pending typo

File: CHANGELOG/v1.9.0.md (hunk 1 above)
Comment (chandankumar4) [reply CHANGELOG/v1.9.0.md]: ohh! might be it got left while updating

**Review by Sunnatillo — APPROVED** [review]
/lgtm

**Pull request merged**